actions/class.ItemContent.php

```diff
@@ -266,9 +246,38 @@ protected function getMediaSource($urlPrefix, $item, $itemLang)
         return $asset->getMediaSource();
     }
 
+    private function buildFilters(array $params): array
```

Comment:
should this be extracted?

Reply:
I think that helped to improve readability